src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -183,7 +159,6 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
               </Button>
               <Button
                 variant="contained"
-                disabled={state.processing || !file}
```

Comment:
Still need this to work. Without it, you can click upload as soon as you've selected a document type and it errors in console.  We want it so user can't try until we've got the bare minimum we need (doctype and an uploaded file.)

Reply:
Fixed this.